### ISSUE TITLE
Update bot to use state: triaged label

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-bots#15]
Having re-organized all the labels on addons repos it would be good to update this bot and waffle to use the `state: triaged` label instead of `triaged`.  Once that's been done we can move all issues under 'triaged' -> `state: triaged.

**Comment by eviljeff** [comment]
does [waffle.io](https://waffle.io/mozilla/addons) think `triaged` is a special label though?

**Comment by tofumatt** [comment]
> does waffle.io think triaged is a special label though?

You can change that in waffle's settings I believe.

**Comment by muffinresearch** [comment]
Yep, waffle can just be updated to use the right label once the bot is doing the right thing.
